[DOCS] Update search_after section with an example (#89328)
elastic/elasticsearch: docs/reference/search/search-your-data/paginate-search-results.asciidoc

using a set of <<sort-search-results,sort values>> from the previous page.

Using `search_after` requires multiple search requests with the same `query` and

> `sort` values. The first step is to run an intial request. In the following example, 
> we use two fields (i.e `date` and `tie_breaker_id`) to sort the results by:
> [source,js]

Reviewer: ```suggestion
`sort` values. The first step is to run an initial request. The following
example sorts the results by two fields (`date` and `tie_breaker_id`):
[source,js]
```

--------------------------------------------------
GET twitter/_search
{
    "query": {
        "match" : {
            "title" : "elasticsearch"
        }
    },
    "sort": [
        {"date": "asc"},
        {"tie_breaker_id": "asc"}      <1>
    ]
}
--------------------------------------------------
// CONSOLE
// TEST[setup:twitter]
// TEST[s/"tie_breaker_id": "asc"/"tie_breaker_id": {"unmapped_type": "keyword"}/]

<1> A copy of the `_id` field with `doc_values` enabled

> In order to "search after" the current page, we take the values of `date` and `tie_breaker_id` 
> from the `sort` array in the last document in the page, and insert those into the `search_after` 
> array like so:

Reviewer: ```suggestion
The search response includes an array of `sort` values for each hit. To retrieve
the next page of results, repeat the request, take the `sort` values from the
last hit, and insert those into the `search_after` array:
```

[source,js]
--------------------------------------------------
GET twitter/_search
{
    "query": {
        "match" : 
            "title" : "elasticsearch"
        }
    },
    "search_after": [1463538857, "654323"],
    "sort": [
        {"date": "asc"},
        {"tie_breaker_id": "asc"}
    ]
}
--------------------------------------------------
// CONSOLE
// TEST[setup:twitter]
// TEST[s/"tie_breaker_id": "asc"/"tie_breaker_id": {"unmapped_type": "keyword"}/]

> We can repeat this process by updating the `search_after` array every time we retrieve a 
> new page of results. If a <<near-real-time,refresh>> occurs between these requests,

Reviewer: ```suggestion
Repeat this process by updating the `search_after` array every time you retrieve a 
new page of results. If a <<near-real-time,refresh>> occurs between these requests,
```
